test/unit_test/pass_test/test_remove_redundant_reshape.py

```diff
@@ -126,14 +126,48 @@ class RemoveRedundantReshapePattern3Test(SinglePassValueTest):
     def test_pass(self):
         self.setup(RedundantReshapePattern3())
 
-        if Version(torch.__version__) <= Version("2.6.0.dev20241015"):
-            self.run_value_test(ConvertLayoutOpToReshape())
-
```

Comment:
This pass should be run after `run_decompositions()`.

Reply:
I added a `run_decompositions()` line because `aten.softmax.int` is decomposed to `aten.softmax.default`.

But, at the same time, `run_decompositions()` may replace some reshape ops to views. Therefore, I just pulled down the position of `ConvertLayoutOpToReshape` call. Which would replace all view ops to reshape ops.